plugins/catalog-backend-module-ldap/config.d.ts

```diff
@@ -56,6 +56,8 @@ export interface Config {
       tls?: {
         // Node TLS rejectUnauthorized
         rejectUnauthorized?: boolean;
+        keys: string;
```

Comment:
looks like this is going to be breaking for people running with `rejectUnauthorized` today. Can we perhaps put them in a sub section or make them optional?

Reply:
@jhaals I am not much familiar with NodeJs, but I will try my best for writing tests !